src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { Link, useNavigate } from 'react-router-dom'
+import '../Introduce/Introduce.scss'
```

Comment:
```suggestion
import './Introduce.scss'
```
같은 위치에 있기 때문에 이렇게 경로를 설정할 수 있겠습니다!

Reply:
경로 확인해서 수정하였습니다. 감사합니다.